source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -173,92 +189,125 @@ that matches your query, use the filtered positional operator
 The filtered positional operator ``$[<identifier>]`` specifies the
 matching array elements in the update document. To identify which array
 elements to match, pair this operator with ``<identifier>`` in an
-``options.arrayFilters`` object.
-
-The ``<identifier>`` term is a placeholder value you assign that represents
-an element of the array field name that prefixes it. This value must
-start with a lowercase letter and contain only alphanumeric characters.
+``arrayFilters`` object.
+
+The ``<identifier>`` value is a placeholder you assign that represents
+an element of the array field. This value must start with a lowercase
+letter and contain only alphanumeric characters.
+
+Usage
+`````
+
+Include a query, an update document, and
+optionally, an options object as parameters in an update operation.
```

Comment:
Suggestion:

I think users that know what components comprise an update operation would benefit from the "update operation" positioned earlier in the sentence as it's otherwise more difficult to follow, and maybe repetitive. 

For example:
"ham, cheese, mustard, buttered bread, and optionally pickles are included in a Croque Monsieur" is correct, but not as easy to read as:
"A Croque Monsieur is made of ham, cheese, ..."

"You can include this operator in an update operation, which includes parameters for a query, update document, and an options object."